server/routes/api.js

```diff
@@ -291,5 +291,9 @@ router.post('/setup', (req, res) => {
 });
 
 module.exports = {
-  router
+  router,
+  paths,
```

Comment:
Can you please put all of these under a separate file of common server utilities?